apps/rush-lib/src/scripts/ci-rush.ts

```diff
@@ -134,8 +134,9 @@ if (!installedVersionValid || installedVersion !== expectedVersion) {
         const environmentVariables: string[] | null = line.match(/\$\{([^\}]+)\}/g);
         let lineShouldBeTrimmed: boolean = false;
         if (environmentVariables) {
-          for (const environmentVariable of environmentVariables) {
-            if (!process.env[environmentVariable]) {
+          for (const token of environmentVariables) {
+            const environmentVariableName: string = token.substring(2, token.length - 1);
```

Comment:
FYI the reason your RegExp is failing to match the capture group is because of the impoverished `String.match()` shorthand.  Whereas `RegExp.exec()` is the general-purpose API that can find multiple matches, and return all the capture groups for each match.

Reply:
`match` is fine if you say `/pattern/g`. `exec` doesn't seem to find multiple matches.